examples/Makefile.am

```diff
@@ -13,8 +13,8 @@ AM_CFLAGS = -I$(top_builddir)/include -I$(top_srcdir)/include \
             -I$(top_srcdir)/include/coap$(LIBCOAP_API_VERSION) \
             $(WARNING_CFLAGS) $(DTLS_CFLAGS) -std=c99
 
-# etsi_iot_01 and tiny are missing
-bin_PROGRAMS = coap-client coap-server coap-rd
+#
+bin_PROGRAMS = coap-client coap-server coap-rd coap-etsi_iot_01 coap-tiny
```

Comment:
I wonder if we really want to have these installed?

Reply:
I wanted to get them built to catch any API updates in the future, but not for them to get installed.  My libtools knowledge is limited and could not think of a way to do this other than include them in bin_PROGRAMS.  I did prefix the names to prevent other application name clashes. 

Easy just to remove them out of the programs list if you want.

Reply:
Maybe a compromise change could be
````
@@ -13,8 +13,9 @@ AM_CFLAGS = -I$(top_builddir)/include -I$(top_srcdir)/include \
             -I$(top_srcdir)/include/coap$(LIBCOAP_API_VERSION) \
             $(WARNING_CFLAGS) $(DTLS_CFLAGS) -std=c99
 
-# etsi_iot_01 and tiny are missing
+#
 bin_PROGRAMS = coap-client coap-server coap-rd
+check_PROGRAMS = coap-etsi_iot_01 coap-tiny
 
 coap_client_SOURCES = client.c
 coap_client_LDADD =  $(DTLS_LIBS) $(top_builddir)/.libs/libcoap-$(LIBCOAP_NAME_SUFFIX).la
````
and so a 'make check' would confirm things are good